tests/test_monkeypatch.py

```diff
@@ -61,6 +63,56 @@ def test_content_renderer(self):
             CMSToolbar(request).content_renderer.__class__, VersionContentRenderer
         )
 
+    def test_prefetch_versioned_m2m_objects(self):
```

Comment:
These tests are not testing the monkey patches, they are testing plugin rendering. Please move these test out of this file.

Reply:
I will move to a separate test_** file